### DISCUSSION TITLE
Buttons inverse in Movistar

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#829] [Proposals]
Había un debate abierto sobre los botones en contenedores inverse de Movistar con la llegada de la nueva marca.

Cómo deberían pintarse los botones en fondos inverse?
La lógica actual de Mística es que sobre fondos de color existe la variante inverse del botón que en todas las marcas es poner los botones en blanco
![image](https://user-images.githubusercontent.com/6722153/193019482-02a53f3a-aae5-4601-bff5-850618daa7d3.png)

Para la nueva marca se discutía el usar botones blancos en fondos oscuros porque funciona bien los azules sobre este fondo. El problema venía en que si se usaba la versión "no inverse" en fondos oscuros, el buttonSecondary tb debía ser azul por lo que la legibilidad se comprometía mucho.

Se ha llegado a un pre-acuerdo y una vía por la que tirar en la que en Movistar los botones inverse se comportará de la siguiente manera:
- PrimaryButton → Azul
- SecondaryButton → Blanco
- ButtonLink → Blanco (+ chevron solo desktop)

a falta de hacer pruebas y confirmar que funcione todo correctamente

**Comment by AnaMontes11** [comment]
Añado ejemplo visual de lo que se plantea
![INVERSE](https://user-images.githubusercontent.com/88187691/194555543-d18121ec-6895-471e-9d81-93cdfcf738ea.jpg)

**Reply by aweell** [reply]
+1 A la propuesta

**Reply by yceballost** [reply]
entiendo que los estados hover no se alteran. Podemos confirmar esto?

**Reply by AnaMontes11** [reply]
Se ha actualizado el hover en los Primary pero el resto de hover se mantienen igual.
